README.md

```diff
@@ -215,16 +215,14 @@ library my_library;
 
 A file `categories.json` will be generated at the top level of the documentation tree with
 information about categories collected from objects in the source tree.  The directives
-`@category`, `@subCategory`, and `@image` are understood and saved into this json.
-Future versions of dartdoc may make direct use of the image tags.
+`@category`, and `@subCategory` are understood and saved into this json.
 
 As an example, if we document the class Icon in flutter using the following:
 
 ```dart
 /// {@category Basics}
-/// {@category Assets, Images, and Icons}
-/// {@subCategory Information displays}
-/// {@image <image alt='' src='/images/catalog-widget-placeholder.png'>}
+/// {@category "Assets and Icons"}
```

Comment:
Oh sorry, that comment was aspirational. The current syntax would not parse the quotes. This should be written without quotes here, so it is parsed OK.

But in the future, I'd like to make a breaking change.

I think I was confused even just reading the text here, the examples, because of the commas. I first thought, "oh can you declare three categories like this?" But the answer is "no," this line (without the quotes) creates a category called "Assets, Images, and Icons".

Reply:
Oh! Woops, my bad. I'll revert the quotes. I agree though in that we should have quotes.